tests/x11/gnomecase/login_test.pm

```diff
@@ -16,11 +16,12 @@ use base "x11test";
 use strict;
 use testapi;
 use utils;
+use version_utils 'is_tumbleweed';
 
 sub auto_login_alter {
     my ($self) = @_;
     $self->unlock_user_settings;
-    send_key "alt-u";
+    send_key "alt-u" unless is_tumbleweed;
```

Comment:
Considering that "tumbleweed" is not really a version and is more like the always changing upstream we want to support I suggest to also never check for this. You can turn around the check by checking e.g.

```perl
send_key 'alt-u' if is_sle('<=15')
```